tests/feature_config.rs

```diff
@@ -18,13 +18,27 @@ impl CargoSemverChecks {
         }
     }
 
+    fn command(&self) -> Command {
+        Command::cargo_bin("cargo-semver-checks").unwrap()
+    }
+
     fn add_arg(&mut self, arg: &str) -> &mut Self {
         self.args.push(String::from(arg));
         self
     }
 
-    fn run(&mut self) -> Assert {
-        self.cmd.args(&self.args).assert()
+    fn run_all(&mut self) -> Vec<Assert> {
+        vec![self.run_without_subcommand(), self.run_with_subcommand()]
+    }
+
+    fn run_without_subcommand(&mut self) -> Assert {
+        let mut args = self.args.clone();
+        args.remove(SUBCOMMAND_ARGS_INDEX);
+        self.command().args(&args).assert()
+    }
+
+    fn run_with_subcommand(&mut self) -> Assert {
+        self.command().args(&self.args).assert()
```

Comment:
Do we need to take `&mut self` on all of these? I'm not sure we're mutating anything on any of them.